tests/datasets_tests/test_empty_dataset.py

```diff
@@ -0,0 +1,29 @@
+import numpy as np
```

Comment:
`unittest` should be the first because it's a standard library, while `numpy` is a third party lib.